lib/bolt/cli.rb

```diff
@@ -564,19 +564,121 @@ def generate_types
       0
     end
 
+    # Initializes a specified directory as a Bolt project and installs any modules
+    # specified by the user, along with their dependencies
     def initialize_project
-      path = File.expand_path(options[:object] || Dir.pwd)
-      FileUtils.mkdir_p(path)
-      ok = FileUtils.touch(File.join(path, 'bolt.yaml'))
+      boltdir    = Pathname.new(File.expand_path(options[:object] || Dir.pwd))
+      config     = boltdir + 'bolt.yaml'
+      puppetfile = boltdir + 'Puppetfile'
+      modulepath = [boltdir + 'modules']
```

Comment:
What consideration do we need to give to the case where a user has a user-level or global `bolt.yaml` that specifies a different `modulepath`?

Reply:
I think it makes the most sense to ignore system-wide and user-level config and have the `project init` command have no concept of the different config levels. 

It does raise an interesting question of what happens with users who have a global or user-level `modulepath` and expect to be able to use the modules in their new project... I think either warning the user that they might need to modify the modulepath in the project config or setting that in the new `bolt.yaml` would be helpful. I lean towards the former.